### PR TITLE
add delegate/undelegate

### PR DESCRIPTION
**Pull request by groninge01**

### CODE REVIEW

File: modules/reliquary/ReliquaryLanding.tsx [hunk 1]
```diff
@@ -69,6 +72,9 @@ export default function ReliquaryLanding() {
     const { isOpen, onOpen, onClose } = useDisclosure();
     const [buttonEnabled, setButtonEnabled] = useState(true);
     const { totalMaBeetsVP, isLoading } = useReliquary();
+    const { data: isDelegatedToMDs } = useDelegation();
+
+    console.log({ isDelegatedToMDs });
```
Comment (mrchickenburger) [inline modules/reliquary/ReliquaryLanding.tsx:77]: rm log

File: modules/reliquary/components/DeligateClearButton.tsx [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+import { useDelegateClear } from '../lib/useDelegateClear';
+import { useDelegation } from '../lib/useDelegation';
+
+interface Props {
+    size?: string;
+    rounded?: string;
+    w?: string;
+}
+
+export default function DeligateClearButton({ ...rest }: Props) {
```
Comment (mrchickenburger) [inline modules/reliquary/components/DeligateClearButton.tsx:14]: You can use 'ButtonProps' which is exported by chakra here

Comment (mrchickenburger) [reply modules/reliquary/components/DeligateClearButton.tsx:14]: Also nit: delegate is not spelt correctly in the components

File: modules/reliquary/components/DeligateClearButton.tsx [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+    const { clearDelegate, ...clearDelegateQuery } = useDelegateClear();
+    const { refetch } = useDelegation();
```
Comment (mrchickenburger) [inline modules/reliquary/components/DeligateClearButton.tsx:16]: Question: Not possible to bundle the delegation hooks into one single hook?

File: modules/reliquary/components/DeligateClearButton.tsx (hunk 2 above)
Comment (groninge01) [reply modules/reliquary/components/DeligateClearButton.tsx:14]: fixed both

File: modules/reliquary/ReliquaryLanding.tsx (hunk 1 above)
Comment (groninge01) [reply modules/reliquary/ReliquaryLanding.tsx:77]: removed

File: modules/reliquary/components/DeligateClearButton.tsx (hunk 2 above)
Comment (groninge01) [reply modules/reliquary/components/DeligateClearButton.tsx:14]: `ButtonProps` somehow meeses with `BeetsSubmitTransactionButtonProps` so reverted it

File: modules/reliquary/components/DeligateSetButton.tsx [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+import { useDelegateSet } from '../lib/useDelegateSet';
+import { useDelegation } from '../lib/useDelegation';
+
+interface Props {
+    size?: string;
+    rounded?: string;
+    w?: string;
+}
+
+export default function DeligateSetButton({ ...rest }: Props) {
```
[the remark this answers is not in the document]
Comment (groninge01) [reply modules/reliquary/components/DeligateSetButton.tsx:16]: See above

File: modules/reliquary/components/DeligateClearButton.tsx (hunk 3 above)
Comment (groninge01) [reply modules/reliquary/components/DeligateClearButton.tsx:16]: I'll add a TODO

**Pull request merged**